frontend_scv/lib/models/contract.dart

```diff
@@ -1,29 +1,30 @@
-import 'dart:convert';
+import 'dart:math';
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
-import 'package:http/http.dart' as http;
 import 'package:unison/models/condition.dart';
 
-import 'http_exception.dart';
+import 'global.dart';
 
 class Contract with ChangeNotifier {
   String contractId; //agreementID
-  BigInt blockchainId; //TODO: remove ples
+  BigInt blockchainId;
   String partyA;
   String partyB;
   DateTime createdDate;
   DateTime sealedDate;
   bool movedToBlockchain;
-  List<Condition> conditions; //TODO Handle empty/initial conditions
+  List<Condition> conditions;
   String title;
 
   String description;
   String payingUser;
-  double price;
+  double paymentAmount; //In UNT
```

Comment:
This causes a precision loss, we will need to address it in the future

Reply:
It's fine if it's only for display purposes, though

Reply:
This allows expressing the amount of currency as a UNT unit, instead of gwei.